spec/system/assessor_interface/checking_submitted_details_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.describe "Assessor check submitted details", type: :system do
+  before do
+    given_the_service_is_staff_http_basic_auth
+    given_there_is_an_application_form
+
+    when_i_am_authorized_as_an_assessor_user
+  end
+
+  it "allows checking the qualifications" do
+    when_i_visit_the_check_qualifications_page
+    then_i_see_the_qualifications
+
+    when_i_click_continue
+    then_i_see_the_application_page
+  end
+
+  private
+
+  def given_the_service_is_staff_http_basic_auth
+    FeatureFlag.activate(:staff_http_basic_auth)
```

Comment:
Should we start testing with this off?

Reply:
Yep good point, I've amended those tests to log in as an assessor user.